src/assets/scss/pages/story.scss

```diff
@@ -25,7 +25,7 @@
   }
 
   &-page {
-    height: 100vh;
+    height: auto;
```

Comment:
this isn't  a good solution either. the `story-page` class is used globally. You will be breaking the UI of other pages with this. The 100vh is to get full vertical hight. Setting it to Auto means it will only get the hight of its container (depending on the content and not the size of the screen)

If the problem is in the footer, you need to work on solutions tackling specifically the component, or footer page.